azure-container-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/containerapps/ConfigMojo.java

```diff
@@ -0,0 +1,432 @@
+     * The prompt wrapper to get user input for each property.
+     */
+    private ConfigurationPrompter wrapper;
+
+    /**
+     * The maven variable used to evaluate maven expression.
+     */
+    @Parameter(defaultValue = "${mojoExecution}")
+    protected MojoExecution mojoExecution;
+
+    private boolean useExistingAppEnv = false;
+
+    @Override
+    @AzureOperation("user/containerapps.config_mojo")
+    protected void doExecute() throws Throwable {
+        if (!MavenConfigUtils.isJarPackaging(this.project)) {
+            throw new UnsupportedOperationException(
+                String.format("The project (%s) with packaging %s is not supported for Azure Container Apps.", this.project.getName(),
+                    this.project.getPackaging()));
+        }
+        if (isProjectConfigured(this.project)) {
+            log.warn(String.format("Project (%s) is already configured and won't be affected by this command.", this.project.getName()));
+            return;
+        }
+        final ExpressionEvaluator expressionEvaluator = new PluginParameterExpressionEvaluator(session, mojoExecution);
+        try {
+            this.wrapper = new ConfigurationPrompter(expressionEvaluator, "container-apps");
+            this.wrapper.initialize();
+            this.wrapper.putCommonVariable("project", this.project);
+
+            // select subscription in container apps
+            // set up account and select subscription here
+            loginAzure();
+            promptAndSelectSubscription();
+
+            // prompt to select existing appEnv or create a new one
+            useExistingAppEnv = this.wrapper.handleConfirm("Use existing Azure Container Apps Environment in Azure (Y/n):", true, true);
+            final ContainerAppsEnvironment appEnv = useExistingAppEnv ? selectAppEnv() : configAppEnv();
+            final boolean useExistingApp = Objects.nonNull(appEnv) &&
+                this.wrapper.handleConfirm(String.format("Use existing app in Azure Container App Environment %s (y/N):", appEnv.getName()), false, true);
+            if (useExistingApp) {
+                selectApp(appEnv);
+            } else {
+                configureAppName();
+            }
+            configCommon();
+            confirmAndSave();
+        } catch (IOException | InvalidConfigurationException | UnsupportedOperationException | MavenDecryptException | AzureToolkitAuthenticationException e) {
+            throw new AzureExecutionException(e.getMessage());
+        } finally {
+            if (this.wrapper != null) {
+                try {
+                    this.wrapper.close();
+                } catch (IOException e) {
+                    // ignore at final step
+                }
+            }
+
+        }
+    }
+
+    private ContainerAppsEnvironment configAppEnv() throws IOException, InvalidConfigurationException {
+        configureAppEnvName();
+        configureResourceGroup();
+        configureRegion();
+
+        return null;
```

Comment:
Why always returns null here? shall we just change to void if the result is not necessary?